### ISSUE TITLE
Error occurred with SVG.text(), v2.7.0, IE11

### ISSUE THREAD

**Issue by mstssk** [svgdotjs/svg.js#938]
# Bug report

Error occurred when I use SVG.text() method on IE11

`Object doesn't support this action` error occurred at this code:
https://github.com/svgdotjs/svg.js/blob/5a72b487125beaa6b7bd81584fb9d35c4ffc43be/src/event.js#L132
[This code modified in v.2.7.0](https://github.com/svgdotjs/svg.js/commit/5a72b487125beaa6b7bd81584fb9d35c4ffc43be#diff-ba5d3e6fd360d6221bb4c0e1aa871a4dR3535)

Workaround: Rollback to v2.6.6.

Minimal example:

```js
const appDiv = document.getElementById('app');
const svg = SVG(appDiv);
svg.text("foobar"); // `text` method breaks on IE11.
```

## Fiddle

https://jsfiddle.net/s06mfv5u/78/

| Screenshot: IE11 on VirtualBox |
|:--------:|
| ![Screenshot](https://user-images.githubusercontent.com/500072/49269178-fcedee00-f4a6-11e8-927b-beb605f6cbda.png) |

## Explanation

- Error don't occurred on IE11.

**Comment by Fuzzyma** [comment]
Sorry, this error was "backported" from the 3.0 branch. I just fixed it and released 2.7.1.

**Comment by Fuzzyma** [comment]
https://github.com/svgdotjs/svg.js/releases/tag/2.7.1

**Comment by mstssk** [comment]
I confirmed this issued is fixed at v2.7.1!
Thanks!
